<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accelerator</class>
 <widget class="QMainWindow" name="Accelerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accelerator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/pic/MainWindow/icon.ico</normaloff>:/img/pic/MainWindow/icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Input">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>600</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Colonna MT</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: white;
	border: 1px solid;
     border-width: 2px;
     border-radius: 12px;
     border-color: grey;
     min-width: 10em;
     padding: 6px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>140</y>
      <width>681</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
     border-width: 2px;
     border-radius: 20px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/img/pic/MainWindow/background.jpg</pixmap>
    </property>
   </widget>
   <widget class="QColumnView" name="text_background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>600</width>
      <height>70</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);
	border style: outset;
     border: 3px solid green;
     border-width: 3px;
     border-radius: 15px;
     min-width: 10em;
     padding: 6px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="keyboard">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>638</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Keyboard.qrc">:/kb/pic/Keyboard/keyboard.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1130</width>
      <height>750</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/img/pic/MainWindow/bg.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>980</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <widget class="QPushButton" name="tutorial">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Обучающий курс слепой печати (необходимо войти)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="toolTipDuration">
        <number>4000</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
       </property>
       <property name="text">
        <string>Обучение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="texts">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Тренировка печати на пользовательских текстах (необходимо войти)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="toolTipDuration">
        <number>4000</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
       </property>
       <property name="text">
        <string>Словари</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RandomText">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Случайные тексты для интерактивного ввода</string>
       </property>
       <property name="toolTipDuration">
        <number>4000</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
       </property>
       <property name="text">
        <string>Случайный текст</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="About">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Информация о программе &quot;Accelerator&quot;</string>
       </property>
       <property name="toolTipDuration">
        <number>4000</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
       </property>
       <property name="text">
        <string>О программе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Quit">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Выход из программы</string>
       </property>
       <property name="toolTipDuration">
        <number>4000</number>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>Quit</zorder>
    <zorder>About</zorder>
    <zorder>RandomText</zorder>
    <zorder>tutorial</zorder>
    <zorder>texts</zorder>
   </widget>
   <widget class="QPushButton" name="Autorization">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Авторизация</string>
    </property>
    <property name="toolTipDuration">
     <number>4000</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-style: outset;
     padding: 6px;
     border: 1px solid #404040;
     border-width: 2px;
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/img/pic/Autorization/avatar.ico</normaloff>:/img/pic/Autorization/avatar.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="current_mistakes">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>220</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Ошибок: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mistakes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Ошибок: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Время: 00:03:3716</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="avg_speed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Скорость: 250</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_speed">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>190</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Скорость: 300</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="past_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Время: 00:03:3716</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="backlight">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>638</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Keyboard/L_shift.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="backlight_shift">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>638</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Keyboard/L_shift.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="intro_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1130</width>
      <height>640</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gold;</string>
    </property>
    <property name="text">
     <string>Accelerator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="intro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1130</width>
      <height>750</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/img/pic/MainWindow/intro.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Text">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>291</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Не откладывай на завтра то, что можно сделать сегодня!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Text_typed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>286</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,200,0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Не откладывай на завтра то, что можно сделать сегодня!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>intro</zorder>
   <zorder>intro_text</zorder>
   <zorder>bg</zorder>
   <zorder>background</zorder>
   <zorder>text_background</zorder>
   <zorder>keyboard</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>Input</zorder>
   <zorder>Autorization</zorder>
   <zorder>current_mistakes</zorder>
   <zorder>mistakes</zorder>
   <zorder>time</zorder>
   <zorder>avg_speed</zorder>
   <zorder>current_speed</zorder>
   <zorder>past_time</zorder>
   <zorder>backlight</zorder>
   <zorder>backlight_shift</zorder>
   <zorder>Text_typed</zorder>
   <zorder>Text</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">     background-color: black;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
  <include location="Keyboard.qrc"/>
 </resources>
 <connections/>
</ui>
